acm/odh-core/acm-observability/grafana-dashboards/edge-inference-health.json

```diff
@@ -188,34 +362,74 @@
           "value": "near-edge-aws-ocp412"
         },
         "datasource": null,
-        "definition": "label_values(rest_server_requests_total, cluster)",
+        "definition": "label_values(cluster)",
         "description": null,
         "error": null,
         "hide": 0,
         "includeAll": false,
-        "label": "Edge Cluster",
+        "label": "",
         "multi": false,
         "name": "cluster",
         "options": [],
         "query": {
-          "query": "label_values(rest_server_requests_total, cluster)",
+          "query": "label_values(cluster)",
           "refId": "StandardVariableQuery"
         },
-        "refresh": 1,
+        "refresh": 2,
         "regex": "",
         "skipUrlSync": false,
-        "sort": 0,
+        "sort": 1,
         "type": "query"
+      },
+      {
+        "allValue": null,
+        "current": {
+          "selected": true,
+          "text": [
+            "All"
+          ],
+          "value": [
+            "$__all"
+          ]
+        },
+        "description": null,
+        "error": null,
+        "hide": 0,
+        "includeAll": true,
+        "label": null,
+        "multi": true,
+        "name": "namespace",
+        "options": [
+          {
+            "selected": true,
+            "text": "All",
+            "value": "$__all"
+          },
+          {
+            "selected": false,
+            "text": "bike-rental-app",
+            "value": "bike-rental-app"
+          },
+          {
+            "selected": false,
+            "text": "tensorflow-housing-app",
+            "value": "tensorflow-housing-app"
+          }
+        ],
+        "query": "bike-rental-app, tensorflow-housing-app",
```

Comment:
It looks like the namespace values still come from a static list here, rather than a query. Does that still need to be changed?

Reply:
Sorry forgot this, it is changed 👍